WalletWasabi/Blockchain/Transactions/TransactionHistoryBuilder.cs

```diff
@@ -42,25 +42,25 @@ public List<TransactionSummary> BuildHistorySummary()
 			}
 			else
 			{
-				var destinationAddresses = containingTransaction.GetDestinationAddresses(wallet.Network, out List<IInput> inputs, out List<Output> outputs);
-				txRecordList.Add(new TransactionSummary(containingTransaction, coin.Amount, inputs, outputs, destinationAddresses));
+				var destinationAddresses = containingTransaction.GetDestinationAddresses(wallet.Network, out _, out _);
```

Comment:
There are 2 references to `SmartTransaction.GetDestinationAddresses()` and both of them discard the 2 `out` parameters.
Why have them if we always discard them?

Reply:
Ah yes, I see now. My suggestion is implemented in Clement's PR. Lovely!